source/application/BioXAS/BioXASSideAppController.cpp

```diff
@@ -21,11 +21,8 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 
```

Comment:
@dretrex What I want to say is that you can start to review from this point forward and ignore the changes beyond this point since they are from Issue1297